NineChronicles.Headless/ActionEvaluationPublisher.cs

```diff
@@ -103,7 +103,10 @@ public async Task AddClient(Address clientAddress)
             if (!_blockRenderDictionary.ContainsKey(clientAddress))
             {
                 _blockRenderDictionary[clientAddress] = true;
-                _blockRenderer.BlockSubject.SubscribeOn(NewThreadScheduler.Default).Subscribe(
+                _blockRenderer.BlockSubject
+                    .SubscribeOn(NewThreadScheduler.Default)
+                    .ObserveOn(NewThreadScheduler.Default)
```

Comment:
We may need other schedulers (e.g., [TaskPoolScheduler](https://learn.microsoft.com/en-us/previous-versions/dotnet/reactive-extensions/hh229933(v=vs.103)) than `NewThreadSheduler`. 🤔